CONTRIBUTING.md

```diff
@@ -123,35 +132,65 @@ cd ocaml && make -j9 world.opt && make install && cd ..
 
 ## Contributing to the runtime
 
-BuckleScript runtime implementation is currently a mix of OCaml and JavaScript. (`jscomp/runtime` directory). The JavaScript code is defined in the `.ml` file using the `bs.raw` syntax extension.
+BuckleScript runtime implementation is currently a mix of OCaml and JavaScript.
+(`jscomp/runtime` directory).
 
-The goal is to implement the runtime **purely in OCaml** and you can help!
+The goal is to implement the runtime **purely in OCaml**, and you can help!
```

Comment:
So the goal here is to make embedded js as less as possible

Reply:
reflected in latest commit